pkgs/by-name/lm/lmstudio/linux.nix

```
@@ -28,7 +31,19 @@ appimageTools.wrapType2 {
     mkdir -p $out/share/applications
     cp -r ${appimageContents}/usr/share/icons $out/share
     install -m 444 -D ${appimageContents}/lm-studio.desktop -t $out/share/applications
+
+    # Rename the main executable from lmstudio to lm-studio
+    mv $out/bin/lmstudio $out/bin/lm-studio
+
     substituteInPlace $out/share/applications/lm-studio.desktop \
-      --replace-fail 'Exec=AppRun --no-sandbox %U' 'Exec=lmstudio'
+      --replace-fail 'Exec=AppRun --no-sandbox %U' 'Exec=lm-studio'
+
+    # lms cli tool
+    install -m 755 ${appimageContents}/resources/app/.webpack/lms $out/bin/
+
+    ${patchelf}/bin/patchelf --set-interpreter "${stdenv.cc.bintools.dynamicLinker}" \
```

```suggestion
    ${lib.getExe patchelf} --set-interpreter "${stdenv.cc.bintools.dynamicLinker}" \
```
(is patchelf not available in the stdenv?)